docgenerator/src/main/java/codacy/codenarc/docs/TypeComparisonHelper.java

```diff
@@ -0,0 +1,7 @@
+package codacy.codenarc.docs;
+
+public class TypeComparisonHelper {
+    public static <A> boolean isInt(Class<A> aClass) {
+        return aClass == int.class;
+    }
+}
```

Comment:
Can you rewrite this file in Scala too?

Reply:
I can't do it in Scala because I need to check if a type is int. And int is not a Scala class.
So, I have to do it in Java